win32ss/include/ntuser.h

```diff
@@ -1244,6 +1244,41 @@ typedef struct tagTRANSMSGLIST
     TRANSMSG TransMsg[ANYSIZE_ARRAY];
 } TRANSMSGLIST, *PTRANSMSGLIST, *LPTRANSMSGLIST;
 
+struct IME_STATE;
```

Comment:
I suppose that including here as well immdev.h in order not to repeat its inclusion in the other files, wouldn't work?
(Alternatively, having a separate win32ss/include/ntimm.h that includes immdev.h plus any private IMM/IME-specific stuff like INPUTCONTEXTDX or other structures related to these, that are already in ntuser.h ?)